src/ui/ui_settings/public/ui_settings_client.js

```diff
@@ -102,6 +102,16 @@ You can use \`config.get("${key}", defaultValue)\`, which will just return
     return this.isDeclared(key) && !('value' in this._cache[key]);
   }
 
+  isControlledByServer(key) {
+    return this.isDeclared(key) && Boolean(this._cache[key].isControlledByServer);
+  }
+
+  assertUpdateAllowed(key) {
+    if (this.isControlledByServer(key)) {
+      throw new Error(`Unable to update "${key}" because it is overridden by the Kibana server`);
```

Comment:
Maybe it's more consistent to say "because it is controlled by the Kibana server"?